src/main/java/org/jitsi/impl/protocol/xmpp/XmppProviderImpl.java

```diff
@@ -108,7 +108,8 @@ public XmppProviderImpl(
 
         connection = createXmppConnection();
         connectRetry = new RetryStrategy(executor);
-        jibriIqHandler = new JibriIqHandler(connection);
+        jibriIqHandler = new JibriIqHandler();
+        connection.registerIQRequestHandler(jibriIqHandler);
```

Comment:
This is awesome! (it working as a handler instead of requiring the connection)

Reply:
Yeah, because it needs to be registered to both providers in order to handle communication with jibri and the clients